[PWPA-2506] Add wellprop to pvt table converter

gabrielantao opened the pull request:
In this PR a converter from welprop tables to pvt table was added.

PWPA-2506

carlosnewmar commented on src/alfasim_score/converter/wellprop/_tests/data/DFLT_FPBNA_OLEO_NACL_10.00/temperature_MIXTURE_conductivity.csv:
Are we using this file in converter? If negative, we can remove all mixture ones.

gabrielantao replied:
I'm not using this, removed.

ThalesCarl commented on README.rst:
[off topic] anyone know we are using .rst instead of .md for the REAME file?

gabrielantao replied:
I also think that is unusual. I think the base structure of repo was created based on some ESSS old repo that used this format.

ThalesCarl commented on README.rst:
```diff
@@ -59,6 +60,11 @@ How to use it
+#. The user also must remember to convert and save the pvt table (as `.tab` file) if wellprop tables are being used
+
+    from alfasim_score.converter.wellprop.wellprop_pvt_table_converter import WellpropToPvtConverter
+    converter = WellpropToPvtConverter(Path("name_of_folder_with_wellprop_tables"))
+    converter.generate_pvt_table_file(Path("name_of_folder_to_save_converted_pvt_table"))
```
The `converter` variable is already being used by the  AlfasimScoreConverter. This might be misinterpreted by the user

gabrielantao replied:
You have a point. Updated. 👍

ThalesCarl reviewed (approved):
Well done!

Commit:
Add the generation of the pvt table buffer

PWPA-2506

Commit:
Remove unecessary wellprop files

Remove old generated reference tabs

PWPA-2506

Commit:
Change the places here reading well strin data

Change thermomechanical_property to plural

PWPA-2506

Commit:
Update the docs for the new converter

PWPA-2506

Commit:
Fix the sonar cloud issues

PWPA-2506

Commit:
Add test data and reference values

PWPA-2506

Commit:
Inclue only python code to the sonar checker

PWPA-2506

Commit:
Generate a pvt dataframe with properties

PWPA-2506

Commit:
Write test to ensure pvt file creating

Put the reader method to return dataframes
Let some methods be private to the converter

PWPA-2506

Commit:
Update name of converters in docs

PWPA-2506
